### DISCUSSION TITLE
Nextcloud AIO Borg Backup time conflict with external backup

### DISCUSSION THREAD

**Discussion by rafgurg** [nextcloud/all-in-one#5135] [Questions] [answered]
If you save the Borg backup to an external NAS (e.g. QNAP) in a Nextcloud AIO instance, you always get an error message during the integrity check because Borg uses UTC, but the NAS uses the regional time (e.g. Europe/Berlin; difference 2 hours) and this conflict is not resolved. A restore from this backup, e.g. to a new Nextcloud AIO instance, unfortunately always fails due to this bug.

### Steps to reproduce
1. Create Borg Backup /mnt/[extern share/NAS]
2. Integrity check
3. Restore Backup

### Expected behavior 
The integrity check should work and it should be possible to use this backup to restore Nextcloud in a new instance. Unfortunately, the time conflict error prevents this.

### Actual behavior 
Error in the logs: Integrity error

### Host OS 
Proxmox VM Ubuntu 22.04

#### Nextcloud AIO version 
Nextcloud AIO v9.4.1

#### Current channel 
latest channel

#### Other valuable info 

I was only able to solve the problem by making a newly initialized backup on a USB stick. Backup from the old VM to the USB stick; then restore to the new VM with a freshly installed Nextcloud AIO instance

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the container logs here?
